src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModelEMWorkspaceGermline.java

```diff
@@ -0,0 +1,102 @@
+import org.broadinstitute.hellbender.tools.exome.germlinehmm.IntegerCopyNumberState;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.GermlinePloidyAnnotatedTargetCollection;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.SexGenotypeData;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.SexGenotypeDataCollection;
+import org.broadinstitute.hellbender.utils.hmm.segmentation.HiddenMarkovModelSegmentProcessor;
+import org.broadinstitute.hellbender.utils.hmm.segmentation.HiddenStateSegmentRecordWriter;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+
+/**
+ * This class represents the driver-node workspace for EM algorithm calculations of the coverage model
+ * specific for germline copy number states.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelEMWorkspaceGermline extends CoverageModelEMWorkspace<IntegerCopyNumberState> {
+
+    /**
+     * Public constructor
+     *
+     * @param rawReadCounts an instance of {@link ReadCountCollection} containing raw read counts
+     * @param germlinePloidyAnnotatedTargetCollection an instance of {@link GermlinePloidyAnnotatedTargetCollection}
+     *                                                for obtaining target ploidies for different sex genotypes
+     * @param sexGenotypeDataCollection an instance of {@link SexGenotypeDataCollection} for obtaining sample sex genotypes
+     * @param params EM algorithm parameters
+     * @param ctx the Spark context
+     * @param copyRatioExpectationsCalculator an implementation of {@link CopyRatioExpectationsCalculator}
+     */
+    public CoverageModelEMWorkspaceGermline(@Nonnull final ReadCountCollection rawReadCounts,
+                                            @Nonnull final GermlinePloidyAnnotatedTargetCollection germlinePloidyAnnotatedTargetCollection,
+                                            @Nonnull final SexGenotypeDataCollection sexGenotypeDataCollection,
+                                            @Nonnull final IntegerCopyNumberExpectationsCalculator copyRatioExpectationsCalculator,
+                                            @Nonnull final CoverageModelEMParams params,
+                                            @Nullable final CoverageModelParameters model,
+                                            @Nullable final JavaSparkContext ctx) {
+        super(rawReadCounts, germlinePloidyAnnotatedTargetCollection, sexGenotypeDataCollection,
+                copyRatioExpectationsCalculator, params, model, ctx);
+    }
+
+    /**
+     * Save copy ratio (here, integer copy number) posteriors to disk
+     *
+     * @param outputPath the output path
+     */
+    protected void saveCopyRatioPosteriors(final String outputPath) {
```

Comment:
all issues have been addressed in the post-PR commit. In fact, we no longer have a `CoverageMModelEMWorkspaceGermline`.